src/test/java/com/coherentsolutions/training/auto/API/pashkovskaya/tests/TestCreatingEmployee.java

```diff
@@ -0,0 +1,27 @@
+package com.coherentsolutions.training.auto.API.pashkovskaya.tests;
+
+import com.coherentsolutions.training.auto.API.pashkovskaya.Employee;
+import com.coherentsolutions.training.auto.API.pashkovskaya.specs.Specs;
+import io.restassured.RestAssured;
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+import static com.coherentsolutions.training.auto.API.pashkovskaya.util.MainConstants.*;
+
+public class TestCreatingEmployee {
```

Comment:
let's unify all the test methods in one class and use dependencies for testng methods